controllers/storagecluster/external_resources.go

```diff
@@ -30,6 +30,7 @@ const (
 	cephFsStorageClassName                = "cephfs"
 	cephRbdStorageClassName               = "ceph-rbd"
 	cephRbdRadosNamespaceStorageClassName = "ceph-rbd-rados-namespace"
+	cephRbdTopologyStorageClassName       = "ceph-rbd-topology"
```

Comment:
can we just use the name from the json blob

Reply:
This is the key we will look for the json blob to get the details of the storageclass, exactly same process as we do for the other storageclasses